src/Controller/Organization/Forecast/PlanningForecastController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Organization\Forecast;
+
+use App\Domain\MissionTypeForecastDomain;
+use App\Domain\PlanningDomain;
+use App\Form\Type\PlanningForecastType;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/forecast", name="forecast", methods={"GET"})
```

Comment:
According to other routes in the project, name should be app_organisation_forecast

Reply:
`planning` is `planning` for now, but I agree :+1: